docs/release_notes.md

```diff
@@ -5,6 +5,125 @@ hide:
+- [[PR #11685]](https://github.com/kubevirt/kubevirt/issues/11685) [fossedihelm] Updated go version of the client-go to 1.21
+- [[PR #11344]](https://github.com/kubevirt/kubevirt/issues/11344) [aerosouund] Refactor device plugins to use a base plugin and define a common interface
+- [[PR #12025]](https://github.com/kubevirt/kubevirt/issues/12025) [fossedihelm] Add descheduler compatibility
+- [[PR #12109]](https://github.com/kubevirt/kubevirt/issues/12109) [acardace] Support Memory Hotplug with Hugepages
+- [[PR #11883]](https://github.com/kubevirt/kubevirt/issues/11883) [orelmisan] Restart of a VM is required when the CPU socket count is reduced
+- [[PR #11655]](https://github.com/kubevirt/kubevirt/issues/11655) [acardace] Allow to hotplug vcpus for VMs with CPU requests and/or limits set
+
+### SIG-storage
+- [[PR #11095]](https://github.com/kubevirt/kubevirt/issues/11095) [ShellyKa13] Expose volumesnapshot error in vmsnapshot object
+- [[PR #11312]](https://github.com/kubevirt/kubevirt/issues/11312) [alromeros] Improve handling of export resources in virtctl vmexport
+- [[PR #11770]](https://github.com/kubevirt/kubevirt/issues/11770) [alicefr] Fix the live updates for volumes and disks
+- [[PR #11957]](https://github.com/kubevirt/kubevirt/issues/11957) [mhenriks] snapshot: Ignore unfreeze error if VMSnapshot deleting
+
+### SIG-network
+- [[PR #11653]](https://github.com/kubevirt/kubevirt/issues/11653) [EdDev] Build the `passt`custom CNI binary statically, for the `passt` network binding plugin.
+- [[PR #11678]](https://github.com/kubevirt/kubevirt/issues/11678) [Vicente-Cheng] Improve the handling of ordinal pod interface name for upgrade
+- [[PR #11618]](https://github.com/kubevirt/kubevirt/issues/11618) [AlonaKaplan] Extend network binding plugin to support device-info DownwardAPI.
+- [[PR #11256]](https://github.com/kubevirt/kubevirt/issues/11256) [andreabolognani] This version of KubeVirt includes upgraded virtualization technology based on libvirt 10.0.0 and QEMU 8.2.0.
+- [[PR #11788]](https://github.com/kubevirt/kubevirt/issues/11788) [ormergi] The network-info annotation is now used for mapping between SR-IOV network and the underlying device PCI address
+- [[PR #11659]](https://github.com/kubevirt/kubevirt/issues/11659) [iholder101] Require scheduling infra components onto control-plane nodes
+- [[PR #12079]](https://github.com/kubevirt/kubevirt/issues/12079) [EdDev] Network hotplug feature is declared as Beta.
+
+### SIG-infra
+- [[PR #11648]](https://github.com/kubevirt/kubevirt/issues/11648) [kubevirt-bot] Updated common-instancetypes bundles to v1.0.0
+- [[PR #11962]](https://github.com/kubevirt/kubevirt/issues/11962) [lyarwood] `VirtualMachines` referencing an instance type are now allowed when the `LiveUpdate` feature is enabled and will trigger the `RestartRequired` condition if the reference within the `VirtualMachine` is changed.
```

Comment:
We can drop this as it was immediately replaced with the actual implementation on the line below, this was just a bugfix we backported to release-1.2 first.